project/Dependencies.scala

```diff
@@ -0,0 +1,33 @@
+import sbt._
+
+object Dependencies {
```

Comment:
why in the separate file? and why the whole dependencies not only versions?

Reply:
> why in the separate file 

Because there are situations when the sub-project has an own `build.sbt` file. Thus I moved this file in the `project` folder.

> and why the whole dependencies not only versions?

It is a standard approach that is used in `sbt new <template>`, for instance.